apps/judicial-system/web/messages/InvestigationCases/Prosecutor/demandsForm.ts

```diff
@@ -34,38 +34,46 @@ export const icDemands = {
           id:
             'judicial.system.investigation_cases:police_demands.demands.prefill.search_warrant',
           defaultMessage:
-            'Í þágu rannsóknar sakamáls er þess krafist að Héraðsdómur Reykjavíkur veiti lögreglustjóranum á höfuðborgarsvæðinu heimild til leitar í {address}, þar sem {accusedName} á dvalarstað, í því skyni að handtaka hann og hafa uppi á munum sem hald skal leggja á. Heimildin nái til leitar í læstum hirslum og í geymslum tilheyrandi framangreindu húsnæði svo og til rannsóknar á efnisinnihaldi raftækja, svo sem símum, tölvum og öðrum rafrænum gagnavörslumunum, sem þar er að finna.',
+            'Í þágu rannsóknar sakamáls er þess krafist að {court} veiti {institution} heimild til leitar í {address}, þar sem {accused} {live, plural, one {býr} other {búa}}, í því skyni að handtaka hann og hafa uppi á munum sem hald skal leggja á. Þess er krafist að heimildin nái til leitar í læstum hirslum og í geymslum tilheyrandi framangreindu húsnæði. svo og til rannsóknar á efnisinnihaldi raftækja, svo sem símum, tölvum og öðrum rafrænum gagnavörslumunum, sem þar er að finna.',
```

Comment:
...eymslum tilheyrandi framangreindu húsnæði. svo og til ran...

Is the . there supposed to be a ,?